src/styles/base/_animated-gradients.scss

```diff
@@ -1,89 +1,109 @@
-@-webkit-keyframes animatedGradient
-{
-  0% { background-position: 100% 0; }
-  100% { background-position: 0 0; }
+@keyframes animatedGradient {
+  0% {
+    background-position: 100% 0;
+  }
+
+  100% {
+    background-position: 0 0;
+  }
 }
-@-moz-keyframes animatedGradient
-{
-  0% { background-position: 100% 0; }
-  100% { background-position: 0 0; }
+@keyframes animatedGradient {
+  0% {
+    background-position: 100% 0;
+  }
+
+  100% {
+    background-position: 0 0;
+  }
 }
-@-o-keyframes animatedGradient
-{
-  0% { background-position: 100% 0; }
-  100% { background-position: 0 0; }
+@keyframes animatedGradient {
+  0% {
+    background-position: 100% 0;
+  }
+
+  100% {
+    background-position: 0 0;
+  }
 }
```

Comment:
we should check browser support and cut the dups

Reply:
Checked caniuse, @keyframes should be supported across the board.